src/table/sorting-model.ts

```diff
@@ -0,0 +1,27 @@
+export class SortingModel {
```

Comment:
It would be nice to keep `SortModel` as simple interface and move `ascCallback` and `descCallback` to util functions (see my comment below).
```typescript
export class SortModel {
  prop: string;
  dir?: SortDirection;
}
```